cmd/up.go

```diff
@@ -368,3 +378,20 @@ func printSvc(w io.Writer, c kubernetes.Interface, nss []string) error {
 	fmt.Fprintln(w)
 	return nil
 }
+
+func allReady(c kubernetes.Interface, nss []string) (bool, error) {
+	for _, ns := range nss {
+		pods, err := c.CoreV1().Pods(ns).List(metav1.ListOptions{
```

Comment:
I would do this with the `Deployments` instead. You could find temporary Pods that are not relevant to know if the installation has finished or not.

Reply:
What kind of temporary pods in the case you mentioned?

Reply:
pods that are being terminated or failed: For example the kubeapps pod will be restarted until the mongodb deployment is ready. You will probably find a time in which both pods are running (and you don't need to wait until the pods that are being terminated are completely removed)

Reply:
Ah I saw your point. Thanks 👍